client/src/components/loginform.js

```diff
@@ -7,14 +7,14 @@ const LoginForm = ()=> {
         <div className="loginform">
             <img src={loginavi} alt="login avi" />
             <h1>Inforum</h1>
```

Comment:
Consider using a form and have the inputs nested in a form. That would be easier for using onSubmit event handler. You can research on forms

Reply:
done